src/main.cpp

```diff
@@ -2919,6 +2912,8 @@ bool ActivateBestChain(CValidationState &state, const CChainParams& chainparams,
 
         CBlockIndex *pindexNewTip = NULL;
         const CBlockIndex *pindexFork;
+        std::list<CTransaction> txConflicted;
+        std::vector<std::tuple<CTransaction,CBlockIndex*,int> > txChanged;
```

Comment:
given the repeated use,  maybe just `typedef` the `std::tuple`?  `tuple` has some nice conveniences over a `struct` alternative.